.changeset/friendly-snakes-sparkle.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Added: `fix` to `Problem` type
```

Comment:
[suggest] This changelog message (also the PR title) seems internal and unhelpful to me.
Instead, what about focusing on the public `report()` API? For example:

```suggestion
Added: `fix`/`unfixable` arguments to `stylelint.utils.report()`
```

Reply:
```suggestion
Added: `fix` callback to the object passed to `stylelint.utils.report()`
```